python/py-rfc3339-validator/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-rfc3339-validator
+version             0.1.4
+revision            0
+distname            rfc3339_validator-${version}
+
+categories-append   devel
+license             BSD
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     38 39 310 311
+python.pep517       no
```

Comment:
what is the issue you see when you don't do this; we should really default to "yes" as is done in the "python" PG.

Reply:
Isn't pyproject.toml required for pep517? Neither of these packages have it.

Reply:
no, that is not strictly needed as it will work fine with `python.pep517` (just tested that locally)